packages/swapper/src/swappers/CowSwapper/getCowSwapTradeQuote/getCowSwapTradeQuote.test.ts

```diff
@@ -382,6 +382,7 @@ describe('getCowSwapTradeQuote', () => {
       allowMultiHop: false,
       slippageTolerancePercentageDecimal: '0.005', // 0.5%
       quoteOrRate: 'quote',
+      originalRate: {} as TradeRate,
```

Comment:
Decided to go with consistently passing `originalRate` for all swappers, despite only Li.Fi requiring it.
Rationale is this is very core data, nothing Li.Fi-specific, and would be quite spaghetty-y to constraint it to Li.Fi, for no benefits.